Source/RSP/RSP Command.c

```diff
@@ -29,7 +29,7 @@
 #include <stdlib.h>
 
 #include "opcode.h"
-#include "RSP.h"
+#include "Rsp.h"
 #include "CPU.h"
 #include "RSP Registers.h"
```

Comment:
Why not leave it as `RSP.h`? Given that other capitalized file names seem to follow that convention (i.e. `CPU.h`, `RSP Registers.h`...)